apps/test/unit/lib/kits/maker/PlaygroundComponentsTest.js

```diff
@@ -84,7 +90,8 @@ describe('Circuit Playground Components', () => {
           let led;
 
           beforeEach(() => {
-            led = createCircuitPlaygroundComponents(board).colorLeds[pin];
+            return createCircuitPlaygroundComponents(board)
+              .then(({colorLeds}) => led = colorLeds[pin]);
```

Comment:
I thought if you had async components in a `beforeEach`, you needed to use the `done` method; is that not the case?